modules/app_lb/https/main.tf

```diff
@@ -10,21 +10,29 @@ resource "google_compute_managed_ssl_certificate" "default" {
 # Configure an HTTPS proxy with the Google-managed certificate and route it to
 # the URL map
 resource "google_compute_target_https_proxy" "default" {
-  name    = "${var.namespace}-https-proxy"
-  url_map = var.url_map.id
-  ssl_certificates = [
-    google_compute_managed_ssl_certificate.default.id
-  ]
+  name             = "${var.namespace}-https-proxy"
+  url_map          = var.url_map.id
+  ssl_certificates = [google_compute_managed_ssl_certificate.default.id]
+  ssl_policy       = google_compute_ssl_policy.ssl-policy.id
```

Comment:
Line 16 will need to be changed to `ssl_policy = google_compute_ssl_policy.default.id` as a consequence of the resource being renamed on line 30